ansible/roles/webhook/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Upload RabbitMQ k8s resource
+  copy:
+    src: "{{ role_path }}/../../../tasks/cockpit-tasks-webhook.yaml"
+    dest: /run/cockpit-tasks-webhook.yaml
+    mode: preserve
+
+# keep this in sync with tasks/run-local.sh
+- name: Generate flat files from RabbitMQ config map
+  shell: |
+    rm -r /etc/rabbitmq
+    mkdir -p /etc/rabbitmq
+    python3 - <<EOF
+    import os.path
+    import yaml
+    with open("/run/cockpit-tasks-webhook.yaml") as f:
+        y = yaml.load(f)
+    files = [item for item in y["items"] if item["metadata"]["name"] == "amqp-config"][0]["data"]
+    for name, contents in files.items():
+        with open(os.path.join('/etc/rabbitmq', name), 'w') as f:
+            f.write(contents)
+    EOF
+
+- name: Clean up existing pod
+  shell: |
+    podman pod rm -f cockpituous || true
```

Comment:
I learned --replace from https://www.redhat.com/sysadmin/podman-features-1 😄

Reply:
Yes, me too -- and the `-l/--last` thing is even better. However, it seems much less clear to me how `--replace` acts for pods, which are "additive" by nature (unlike containers). For scripts it still seems cleaner to me to first clean up the whole old pod rather than trying to replace individual containers in a running one.